rpcs3/Emu/Audio/AudioBackend.cpp

```diff
@@ -100,7 +100,7 @@ void AudioBackend::normalize(u32 sample_cnt, const f32* src, f32* dst)
 	}
 }
 
-std::pair<AudioChannelCnt, AudioChannelCnt> AudioBackend::get_channel_count_and_downmixer(u32 device_index)
+std::pair <AudioChannelCnt, AudioChannelCnt> AudioBackend::get_channel_count_and_downmixer(u32 device_index)
```

Comment:
these changes are still unnecessary and also conflict with the code style used everywhere else ;(

Reply:
Ugh dang it

Reply:
you know, you can just revert lines with git. you don't have to do it manually